test/testenv/verificationutils.go

```diff
@@ -375,3 +384,43 @@ func VerifyIndexerClusterScalingUp(deployment *Deployment, testenvInstance *Test
 		return idxc.Status.Phase
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(splcommon.PhaseScalingUp))
 }
+
+// VerifyStandaloneUpdating ensures Standalone is scaling up
+func VerifyStandaloneUpdating(deployment *Deployment, deploymentName string, standalone *enterprisev1.Standalone, testenvInstance *TestEnv) {
```

Comment:
nit: This can be replaced by code in Scaling Test PR which is a general method to verify phase.